### ISSUE TITLE
--output table-wide never return

### ISSUE THREAD

**Issue by tanvp112** [aws/amazon-ec2-instance-selector#331]
I tried the following command `ec2-instance-selector --memory 4 --vcpus 2 --cpu-architecture x86_64 -r us-east-1 -o table-wide` on Windows 11 with `aws-cli/2.15.11` and the command never return. Eventually the process has to be killed. I have tried the same on Fedora with same aws cli version and the outcome is the same.

Any body has similar issue?

**Comment by elementalvoid** [comment]
I experience this too. It happens on both `table-wide` and `interactive`.

Thing is, it _does work_ it's just excruciatingly slow the first time. Once the cache is built it seems to be reasonably fast.

Nuking the cache and running took 2 minutes. Subsequent runs took only 6-8 seconds.

The `us-east-1-spot-pricing-cache.gob` cache file seems to be what takes all the time. Interestingly, specifying `--usage-class on-demand` on the first run after nuking the cache _still_ results in that cache file being generated even though the requested filter won't select spot instances. `¯\_(ツ)_/¯`

**Comment by tanvp112** [comment]
Yes, I can confirm the same command now return result as expected. Thanks!
